etc/chromeos/lib/installs.sh

```diff
@@ -32,6 +40,9 @@ install_docker() {
     echo "Install docker"
     curl -sSL https://get.docker.com/ | sh
     sudo usermod -a -G docker "$USER"
+
+    echo "Install docker-compose"
+    pip3 install docker-compose
```

Comment:
We should probably be installing docker-compose via curl, instead of via pip since docker-compose version 2 was rewritten in golang